.github/workflows/report-stable.yml

```diff
@@ -25,13 +28,10 @@ jobs:
           _results/
         retention-days: 30
       if: "!cancelled()"
-    #### Disabled action ####
-    # - name: File Issue
-    #  uses: JasonEtco/create-an-issue@v2
-    #  env:
-    #    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    #  with:
-    #    filename: _results/issue.md
-    #    update_existing: true
-    - name: Please file an issue
-      run: echo "Dear Human, Please file an issue with the following template"; cat _results/issue.md
+    - name: File a GitHub Issue
+      run: |
+        gh issue create --title "$TITLE" --body "$(cat _results/body.md)"
```

Comment:
https://cli.github.com/manual/gh_issue_create
Says that the option -F/--body-file can be used to read a file for the body. That avoids some minor shell acrobatics.

Reply:
Cool, updated with `--body-file`.